libbeat/outputs/shipper/config.go

```diff
@@ -53,6 +53,7 @@ type Config struct {
 func defaultConfig() Config {
 	enabled := true
 	return Config{
+		// agent will expect that TLS is enabled by default, will disable explicitly
```

Comment:
> will disable explicitly

Looks confusing because of the `enabled := true` above. What does it mean?

Reply:
Ah, sorry, it should say "the agent will disabled it explicitly"